### ISSUE TITLE
[TorchInductor] `efficient_conv_bn_eval_graph_transform_inlined` miscompiles Conv-BN when convolution arguments are passed by keyword

### ISSUE THREAD

**Issue by miyorena** [pytorch/pytorch#196165]
### 🐛 Describe the bug

`efficient_conv_bn_eval_graph_transform_inlined` reads convolution arguments only from `conv_node.args`. If `F.conv2d` passes arguments such as `bias`, `stride`, or `padding` by keyword, they are dropped by the rewrite, causing incorrect results.

## Minimal reproducer

```python
from __future__ import annotations

import torch
import torch.nn.functional as F
from torch._inductor import config
from torch._inductor.fx_passes.pre_grad import efficient_conv_bn_eval_pass
import torch._inductor.fx_passes.efficient_conv_bn_eval  # noqa: F401


class Model(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.weight = torch.nn.Parameter(torch.randn(4, 3, 3, 3))
        self.bias = torch.nn.Parameter(torch.randn(4))
        self.register_buffer("running_mean", torch.randn(4))
        self.register_buffer("running_var", torch.rand(4) + 1.0)

    def forward(self, x: torch.Tensor) -> torch.Tensor:
        conv = F.conv2d(
            x,
            self.weight,
            bias=self.bias,
            stride=2,
            padding=1,
        )
        return F.batch_norm(conv, self.running_mean, self.running_var)


torch.manual_seed(1703)
model = Model().eval()
x = torch.randn(2, 3, 16, 16)
gm = torch.fx.symbolic_trace(model)
with torch.no_grad():
    expected = gm(x)

with config.patch(
    {
        "efficient_conv_bn_eval_fx_passes": True,
        "freezing": False,
    }
):
    efficient_conv_bn_eval_pass.apply(gm.graph)

gm.graph.lint()
with torch.no_grad():
    actual = torch.fx.Interpreter(gm).run(x)

print("expected shape:", tuple(expected.shape))
print("actual shape:  ", tuple(actual.shape))
assert expected.shape != actual.shape or not torch.allclose(
    actual, expected, rtol=1e-5, atol=1e-5
)
```

## Print Result

```text
expected shape: (2, 4, 8, 8)
actual shape:   (2, 4, 14, 14)
```

### Versions

PyTorch revision `cf30153`.

cc @chauhang @penguinwu @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @kadeng @muchulee8 @amjames @aakhundov @coconutruben @jataylo
